_episodes/14-supplemental-vscode.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Supplemental: Using Git from VS-Code"
```

Comment:
Can the title be more explanatory? Perhaps: "Using Git with an Integrated Development Environment (Visual Studio Code)". Best not to assume that participants are familiar with VS-Code or the difference between IDEs and text editors.